openwisp_controller/config/static/config/js/widget.js

```diff
@@ -153,6 +153,29 @@
         return error;
     };
 
+    var validateMaxLength = function(){
+        var config_variables = {};
+        if (window.getContext()){
+            config_variables = JSON.parse(window.getContext().value);
+        }
+        var config_variables_keys = Object.keys(config_variables);
+        var all_fields = $(".jsoneditor .vTextField");
+        all_fields.map((index, field)=>{
```

Comment:
I am not sure why the JS linter is not complaining (that's really a bummer) but can you please follow the same style of the rest of the document? Laving a blank space between curly braces and so on? Take a look at the rest of the file and follow the same conventions please.